packages/docs/jest.config.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
```

Comment:
I printed the config from `npm run test -- --debug` then inlined it here in a mostly minimal way